app/views/admin/bulk_assignments/new.html.erb

```diff
@@ -1,36 +1,37 @@
+<% content_for :body_layout_body_color, 'white' %>
```

Comment:
if this is gonna be every page, do we need to have it in a `content_for`? seems like we'd want to use scss color variables, so it would have to live with the rest of the scss.